### PR TITLE
Feature/mc 10400 get release details

### PR DESCRIPTION
**Pull request by evanluc**
Documentation for get releases. 

Should be merged after https://github.com/cloudops/cloudmc-api-docs/pull/81 (this branch was not created off of `cmc-dev` so the diff looks huge right now but after merge it will be fine.)

### CODE REVIEW

File: source/includes/gcp/_k8_releases.md [hunk 1]
```diff
@@ -143,5 +143,175 @@ Attributes | &nbsp;
...
+        ],
+        "icon": "https://s3-us-west-1.amazonaws.com/aerospike-fd/wp-content/uploads/2016/06/Aerospike_square_logo.png",
+        "apiVersion": "v1",
+        "appVersion": "v4.5.0.5",
+        "deprecated": false
+      },
+      "templates": [
+        {
+          "name": "templates/NOTES.txt",
+          "data": "VGhlIEFlcm9zcGlrZSBjYW4gYmUgYWNjZXNzZWQgdmlhIHBvcnQgMzAwMCBvbiB0aGUgZm9sbG93aW5nIEROUyBuYW1lIGZyb20gd2l0aGluIHlvdXIgY2x1c3RlcjoKCiAge3sgdGVtcGxhdGUgImFlcm9zcGlrZS5mdWxsbmFtZSIgLiB9fS57eyAuUmVsZWFzZS5OYW1lc3BhY2UgfX0uc3ZjLmNsdXN0ZXIubG9jYWwKCllvdSBjYW4gY29ubmVjdCB0byBhZXJvcGlrZSBpbiB5b3VyIGxvY2FsIG1hY2hpbmUgdXNpbmcgcG9ydC1mb3J3YXJkaW5nOgoKICBleHBvcnQgUE9EX05BTUU9JChrdWJlY3RsIGdldCBwb2RzIC0tbmFtZXNwYWNlIHt7IC5SZWxlYXNlLk5hbWVzcGFjZSB9fSAtbCAiYXBwPXt7IHRlbXBsYXRlICJhZXJvc3Bpa2UubmFtZSIgLiB9fSxyZWxlYXNlPXt7IC5SZWxlYXNlLk5hbWUgfX0iIC1vIGpzb25wYXRoPSJ7Lml0ZW1zWzBdLm1ldGFkYXRhLm5hbWV9IikKICBrdWJlY3RsICAtLW5hbWVzcGFjZSB7eyAuUmVsZWFzZS5OYW1lc3BhY2UgfX0gcG9ydC1mb3J3YXJkICRQT0RfTkFNRSAzMDAwOjMwMDAK"
```
Comment (evanluc) [inline source/includes/gcp/_k8_releases.md]: Not sure if we want to include this base64 encoded data. Thoughts?

Comment (fyrnaga) [reply source/includes/gcp/_k8_releases.md]: trunc them a bit to make shorter and invalid. We don't want to have them release decode

File: source/includes/kubernetes/_k8_releases.md [hunk 2]
```diff
@@ -136,7 +136,171 @@ Attributes | &nbsp;
 `version`<br/>*string* | The revision of the release
 `namespace`<br/>*string* | The namespace to which the release is installed
 
-The information is not totally returned in the list. We filter out the manifest portion. We also filter ou  returning the files and the templates of the charts details. This information will be present in the get request for an individual release.
+The information is not totally returned in the list. We filter out the manifest portion. We also filter out returning the files and the templates of the charts details. This information will be present in the get request for an individual release.
 
+### Get release
```
Comment (fyrnaga) [inline source/includes/kubernetes/_k8_releases.md:141]: Apply the same comments as above

Comment (evanluc) [reply source/includes/kubernetes/_k8_releases.md:141]: Truncated!

**Review by fyrnaga — APPROVED** [review]
LGTM

File: source/includes/kubernetes/_k8_releases.md [hunk 3]
```diff
@@ -136,7 +136,171 @@ Attributes | &nbsp;
 `version`<br/>*string* | The revision of the release
 `namespace`<br/>*string* | The namespace to which the release is installed
 
-The information is not totally returned in the list. We filter out the manifest portion. We also filter ou  returning the files and the templates of the charts details. This information will be present in the get request for an individual release.
+The information is not totally returned in the list. We filter out the manifest portion. We also filter out returning the files and the templates of the charts details. This information will be present in the get request for an individual release.
```
Comment (GuillaumeVialle) [inline source/includes/kubernetes/_k8_releases.md]: "get request" -> "GET request"

**Review by GuillaumeVialle — APPROVED** [review]
LGTM. See comment above.

File: source/includes/kubernetes/_k8_releases.md (hunk 3 above)
Comment (evanluc) [inline source/includes/kubernetes/_k8_releases.md]:
```suggestion
The information is not totally returned in the list. We filter out the manifest portion. We also filter out returning the files and the templates of the charts details. This information will be present in the GET request for an individual release.
```

File: source/includes/gcp/_k8_releases.md [hunk 4]
```diff
@@ -143,5 +143,175 @@ Attributes | &nbsp;
 
 The information is not totally returned in the list. We filter out the manifest portion. We also filter ou  returning the files and the templates of the charts details. This information will be present in the get request for an individual release.
```
Comment (evanluc) [inline source/includes/gcp/_k8_releases.md]:
```suggestion
The information is not totally returned in the list. We filter out the manifest portion. We also filter ou  returning the files and the templates of the charts details. This information will be present in the GET request for an individual release.
```

**Pull request merged**